app/webpacker/packs/application.js

```diff
@@ -9,6 +9,14 @@ new CookieMessage($cookieMessage).init();
 
 var $copyWarningMessage = document.querySelector('[data-copy-course="warning"]');
 if ($copyWarningMessage) {
+  var $enrichmentForm = document.querySelector('[data-qa="enrichment-form"]');
+
+  if ($enrichmentForm) {
```

Comment:
This seems weird... would it not work if we moved the call below to `window.onbeforeunload` to inside an `if` instead of setting it to null on submit?

```js
if (!$enrichmentForm) {
  window.onbeforeunload = function() {
	  return 'You have unsaved changes, are you sure you want to leave?'
	}
}

Reply:
I've realised the logic is slightly different after it was pointed out irl, disregard 👍